book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] Dear @MNisarAli, I notice you have  `book.rentals << self` while you initialize rentals, But it's mandatory to add a method to add new rentals to the book. Please consider adding a `add_rental(person, date)` method to add a rental to the current book. [More here about both-way relationship](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)

Reply:
Hi @TadesseDev 👋, I am grateful to you for highlighting the need to incorporate the compulsory `add_rental` method. Consequently, I have included the `add_rental(person, date)` method, which will help in adding rentals for the current book.

Reply:
Good job @MNisarAli 👌